### DISCUSSION TITLE
Linking to other parts of a pdf

### DISCUSSION THREAD

**Discussion by Ragepanda** [diegomura/react-pdf#1397] [Q&A]
I've been having trouble identifying if it's possible to use a Link component to navigate to a different part of the PDF, like using hashtags to navigate to specific element ids for a standard web page. Does anyone know if this is possible?

**Comment by Vatsal2000** [comment]
@Ragepanda Hi, were you able to find a solution or workaround for this. I need the same functionality.

**Comment by CoskunCakir** [comment]
I don't know if you can use link in this case but you can use bookmarks to navigate inside a pdf, see below:
https://react-pdf.org/advanced#bookmarks
